content/resources/crownlabs_ssh/crownlabs_ssh.md

```diff
@@ -0,0 +1,97 @@
++++
+fragment = "content"
+#disabled = true
+date = "2021-11-12"
+weight = 100
+# title = "Connecting to CrownLabs with SSH"
++++
+
+## Introduction
+
+[**CrownLabs**](https://crownlabs.polito.it/) allows users access via SSH to their VMs using SSH directly on their machines.
+
+In this guide you are going to learn how to generate a key on the main operative systems (Windows, Linux, MacOS), how to load it on [**CrownLabs**](https://crownlabs.polito.it/) and log in your [**CrownLabs**](https://crownlabs.polito.it/)'VM using SSH from terminal.
+
+## How to generate an SSH key pair
```

Comment:
Would add also here at the beginning a comment that this process needs to be done only once. Most likely, no one will ever read the last sentence.